src/include/non-scalar/hubbard_v.js

```diff
@@ -0,0 +1,7 @@
+import { Property } from "../../property";
+
+export class HubbardVProperty extends Property {
+    get values() {
```

Comment:
We can define PropertyWithValues and just inherit all HubbardVProperty, VNN, U from it